terraform/loadbalancer.tf

```diff
@@ -0,0 +1,137 @@
+resource "aws_lb" "keyconjurer" {
+  name_prefix = "keycon"
+  internal    = true
+  subnets = var.subnets
+  security_groups = concat(var.lb_security_group_ids, [
+    aws_security_group.keyconjurer-lb.id
+  ])
+}
+
+resource "aws_lb_listener" "keyconjurer" {
+  load_balancer_arn = aws_lb.keyconjurer.arn
+  # TODO: HTTPS
+  # TODO: redirect all http to https
```

Comment:
Are you going to fix it in this PR, do we have a followup ticket for that?

Reply:
I will fix it in this PR. I put this PR down when doing other stuff and forgot to come back to this.